docs/source/private/overview.rst

```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
 You can open and edit the file using e.g. VIM or Microsoft Word. Please fill out this form for each private dataset and upload
 the file (as a pdf or text file) together with the other results in your zenodo upload.
 
+.. _private-data-gather:
 
 How will this be gathered?
 ==========================
 
-.. _private-data-gather:
-
 We anticipating gathering the data through:
 
 1. A script provided by the OpenFE team that will extract relevant information from your simulations.
 2. A text file for you to fill in with information.
 3. A CSV file with experimental data that you will need to prepare.
 
+To prepare the results for the submission, please follow these steps:
+
+1. Run the :ref:`post-simulation cleanup` script.
+2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
```

Comment:
Maybe add that the contents are all human-readable and should be reviewed before sharing?

Reply:
Added this!